README.md

```diff
@@ -10,6 +10,7 @@ Welcome to the open source project for the Sitecore Migration app. The purpose o
 - Cypress 16.6.0
 - Sitecore Content Hub One (to drive the data for the application)
 - Sitecore CDP/Personalize (not required)
+- Azure Table Storage (either using the emulator or a live account)
```

Comment:
I dug into this today and although I can show a more friendlier Server side error in the console, it still errors like this in the client.  It's complicated because I basically need to handle the configuration error in AzureTableService, to be then handled gracefully by the Next.js api, and then have the AzureProxyService handle that error gracefully.

I think I should push back on this fix.  I'll keep the server side console error in place, and would be open to fixing this, but it'll require alot of changes, so might be better as a tech debt future item.

Reply:
![image](https://github.com/Sitecore/Sitecore-Migration-Game/assets/5227854/e60c5de2-a56d-4060-a8a5-587a1e5d686f)

That is the new error, keeping in mind, this only occurs server side in your terminal, you wouldn't see this error in chrome, because those environment variables are server side only.  (which is why they are a pain to begin with)